src/logics/ArrayTheory.cc

```diff
@@ -8,18 +8,15 @@
 
 #include "ArrayHelpers.h"
 #include "DistinctRewriter.h"
-#include "TreeOps.h"
 
-
-bool ArrayTheory::simplify(vec<PFRef> const & formulas, PartitionManager &, int curr)
-{
+PTRef ArrayTheory::simplifyTogether(const vec<PTRef> & assertions, bool) {
```

Comment:
Why do we need bool param if it is never used?

Reply:
That parameter is used only in `UFTheory` now. It is related to handling `distinct` operations.
There should be a better mechanism for that, but I don't know yet how that should look like.